src/main/java/com/investmenttracker/enterprise/dto/investment.java

```diff
@@ -2,9 +2,19 @@
 
 import lombok.Data;
 
+/**
+ * Represents investment data
+ * id: Investment ID
+ * symbol: Interactive Ticker
+ * shares: Number of Shares
+ * priceOpened: Price that the share opened
+ * priceClosed: Price that the share closed
+ * openedTimestamp: Timestamp for the stock market open
+ * closedTimestamp: Timestamp for the stock market close
+ * profit: Profit from shares
+ */
```

Comment:
Added JavaDoc above investment class to describe the purpose of each of the variables. This is beneficial to code reviewers and collaborators.

Documentation is important! Read this article to view more about how to properly document code: https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
I was pleased to see that the naming convention of the variables for this project abides by clean code standards. Good job on those!
https://www.baeldung.com/java-clean-code

Reply:
Great Javadoc, as it is complete and descriptive!